crowbar_framework/lib/crowbar/backup/export.rb

```diff
@@ -128,6 +129,15 @@ def crowbar
         end
       end
 
+      def batch
+        logger.debug "Backing up batch export"
```

Comment:
doesn't make sense here as this is just a substep of the backup creation. and logging the path is already done at the beginning of the backup creation in https://github.com/crowbar/crowbar-core/blob/master/crowbar_framework/app/models/backup.rb#L160